### ISSUE TITLE
[Accessibility]A11y_AzureToolsStorageExplorer_ManageAccessPolicies_Add : AI4D: (StorageExplorer/ 'Expiry time') The LocalizedControlType property must not be null.

### ISSUE THREAD

**Issue by Radha2019** [microsoft/AzureStorageExplorer#1879]
**“Check out Accessibility Insights! - Identify accessibility bugs before check-in and make bug fixing faster and easier.”**

GitHubTags: #A11y_AzureToolsStorageExplorer; #A11yMAS; #A11YTCS; #DesktopApp; #MAS4.1.2.; #AI4D; #Win10; 

**Environment Details:**
Windows Version: Windows10

**Steps to Reproduce:**
1. Launch Storage Explorer
2. Navigate to "Open Connect dialog" pane and Sign in to Azure.
3. Go to "Manage Accounts" pane and Select the subscriptions you will use. Once selected, click on "Apply" button.
4. Navigate to "Blob Containers" of the selected subscription, right click and Select "Create Blob Container" and hit enter.
5. Right Click on newly created blob container and select "Manage Access Policies" and hit enter.
6. Navigate to "Add" control and hit enter.
7. Verify that all the elements of "Add" controls are accessible and MAS Compliant.

**Actual:**
(StorageExplorer/ 'Expiry time') The LocalizedControlType property must not be null.

**Same issue is observed in below Scenarios:**
1. **Application:**
* Here in this Scenario Localized control type does not exist for controls(Application, Data Explorers, Transfers).

**Expected:**
(StorageExplorer/ 'Expiry time') The LocalizedControlType property must not be null. it should be provided.

**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
https://microsoft.sharepoint.com/teams/msenable/mas/Pages/browse-fixes.aspx

**User Impact**: Screenreader user unable to know the name of the check box.

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={248054a6-5e68-4771-9e1e-242fb5025730} 

**Attachment for Reference**

![1879_AccessibilityA11y_AzureToolsStorageExplorer_Manage Access Policies_Add  AI4D](https://user-images.githubusercontent.com/55691739/65419709-e42b3c00-de1c-11e9-8684-1f92d16d2932.png)

**Comment by craxal** [comment]
HTML elements look fine, so I'm guessing this property just isn't defined for this kind of control. However, the individual parts (year, month, etc.) are identified as spin buttons. One could argue that the `datetime-local` control is a compound control and doesn't actually need these properties, because focus goes to the spin buttons inside the control.

**Comment by Suraj1200499** [comment]
@craxal for datetime-local control NVDA is reading the control type as edit when NVDA focus goes to Start time control and also NVDA announce spin button for date, month and year this issue works fine.

As mentioned in above bug still issue repros in below screen:
**Scenario: Application**
For the Screen "Settings", for controls(Application, Data Explorers, Transfers) NVDA and JAWS announces control name and Clickable but not announces control type.
So here controls(Application, Data Explorers, Transfers) needs to be announced with appropriate control type.

**Comment by craxal** [comment]
@Suraj1200499 I'm not able to reproduce. NVDA v2019.2.1 announces the start and expiry controls as expected:

> Start time date edit, Year Start time spin button 2019

**Comment by Suraj1200499** [comment]
@craxal 
Agreed with your comments and I also conveyed in above comments like issue works fine with start and expiry controls as expected. 

But in above bug we have mentioned like below, same issue exist for other controls.
"**Same issue is observed in below Scenarios:**
Application:
Here in this Scenario Localized control type does not exist for controls(Application, Data Explorers, Transfers)."
NVDA and JAWS not announces the control type of controls (Application, Data Explorers, Transfers), announces only like clickable and the name of the control which is incorrect.

For these controls(Application, Data Explorers, Transfers) only issue still exist.

**Comment by craxal** [comment]
@Suraj1200499 Sorry, but "Application, Data Explorers, Transfers" is rather vague, so I don't know what scenarios you're describing or where in the application you're referring to. Please modify your comments to be more specific.

**Comment by Suraj1200499** [comment]
@Craxal  Please check with below repro steps, will be able to get the screen.
"Application, Data Explorers, Transfers" these are Tab controls present under "Settings" Screen.

**Repro Steps**:
1. Navigate to "Settings" control present on the left of the screen below "Open connect dialog" and hit enter.
2. "Settings" Screen gets opened.
3.  Verify Screen reader not announces Control type for controls "Application, Data Explorers, Transfers".

**Comment by craxal** [comment]
@Suraj1200499 That doesn't have anything to do with the Manage Access List dialog or `datetime-local` inputs. This problem in Settings is already reported in #1970, so I'm closing this as unreproducible.

**Comment by Suraj1200499** [comment]
@Craxal The issue of Setting Screen mentioned in bug#[1879](https://github.com/microsoft/AzureStorageExplorer/issues/1879) is for Win32 Platform and the issue #[1970](https://github.com/microsoft/AzureStorageExplorer/issues/1970) is logged for MAC. 
So the issues #1879 and #1970 are logged for different platform.

**Comment by vinodbk197** [comment]
Verified this issue on:Azure Tools Storage Explorer
Application Version: 1.12.0

The issue is no longer to reproduces,Hence closing this bug.
![image](https://user-images.githubusercontent.com/55909580/73660229-c4a2fa80-46bd-11ea-8ca0-83923df31872.png)

**Comment by fsteffi** [comment]
GithubTags:#Closed;

**Comment by mstechie** [comment]
GitHubTags:#BM-TCS-StorageExplorer-Win32-Sep2019;
